### ISSUE TITLE
Fix the bulk documentation

### ISSUE THREAD

**Issue by wernight** [elastic/elasticsearch-php#318]
The last 2 examples regarding Bulk on https://www.elastic.co/guide/en/elasticsearch/client/php-api/2.0/_indexing_documents.html (crud page) seem to have completely wrong code.

I expect the code on #177 to be actually correct.

**Comment by wernight** [comment]
Ok after some fiddling here is a correct piece of code:

1. Create the index
2. Run something like:

````
$params = ['body' => []];
$count = 0;

foreach ($items as $item) {
    $params['body'][] = [
        'index' => [
            '_index' => 'my-index',
            '_type' => 'my-document',
            '_id' => $item->id,
        ]
    ];
        
    $params['body'][] = [
        'title' => item->title,
        'author' => item->author,
        // ...
    ];

    if (++$count >= 1000) {
        // Send a batch of requests
        $client->bulk($params);
        $params = ['body' => []];
    }
}

if (!empty($params['body'])) {
    $client->bulk($params);
}
````

**Comment by polyfractal** [comment]
Fixed in #386.
